dash_bootstrap_components/__init__.py

```diff
@@ -26,4 +29,6 @@
     _component._css_dist = _css_dist
 
 
-from ._components import *  # noqa
```

Comment:
@pbugnion I think I accidentally isorted this to the top of the file, but I'm not totally confident what the significance of the placement here is. Everything appears to work for me, but I'm worried about issues that I might be blind to. Should I put it back at the bottom of the file?